src/main/kotlin/gg/essential/vigilance/gui/SettingsCategory.kt

```diff
@@ -107,7 +107,7 @@ class SettingsCategory(category: Category) : UIContainer() {
 
         scroller.setVerticalScrollBarComponent(scrollBar, true)
 
-        scroller.onMouseScroll {
+        scroller.addScrollAdjustEvent(true) { _, _ ->
```

Comment:
I guess practically it doesn't make a difference cause the ScrollComponent invokes both all the time, but logically it should be `isHorizontal = false`, no?